src/doc_builder/autodoc.py

```diff
@@ -533,36 +533,47 @@ def resolve_links_in_text(text, package, mapping, page_info):
     prefix = f"/docs/{package_name}/{version}/{language}/"
 
     def _resolve_link(search):
-        object_name, last_char = search.groups()
+        object_or_param_name, last_char = search.groups()
         # Deal with external libs first.
-        lib_name = object_name.split(".")[0]
+        lib_name = object_or_param_name.split(".")[0]
         if lib_name.startswith("~"):
             lib_name = lib_name[1:]
         if lib_name in HUGGINFACE_LIBS and lib_name != package_name:
-            link = get_external_object_link(object_name, page_info)
+            link = get_external_object_link(object_or_param_name, page_info)
             return f"{link}{last_char}"
+        object_name, param_name = None, None
         # If the name begins with `~`, we shortcut to the last part.
-        if object_name.startswith("~"):
-            obj = find_object_in_package(object_name[1:], package)
-            object_name = object_name.split(".")[-1]
+        if object_or_param_name.startswith("~"):
+            obj = find_object_in_package(object_or_param_name[1:], package)
+            object_name = object_or_param_name.split(".")[-1]
+        # If "#" is in the name, assume it's a link to the function/method parameter.
+        elif "#" in object_or_param_name:
+            obj = find_object_in_package(object_or_param_name.split("#", 1)[0], package)
+            param_name = object_or_param_name.split("#", 1)[-1]
```

Comment:
This should probably be done before the test at line 546, as it should be possible to do
```
[`~package.module.object#param`]
```